pkg/ebpf/tracee.go

```diff
@@ -1639,9 +1639,11 @@ func (t *Tracee) invokeInitEvents(out chan *trace.Event) {
 
 	emit = t.eventsState[events.ExistingContainer].Emit
 	if emit > 0 {
-		for _, e := range events.ExistingContainersEvents(t.containers, t.config.NoContainersEnrich) {
-			setMatchedPolicies(&e, emit, t.config.Policies)
-			out <- &e
+		existingContainerEvents := events.ExistingContainersEvents(t.containers, t.config.NoContainersEnrich)
+		for i := range existingContainerEvents {
+			event := &(existingContainerEvents[i])
```

Comment:
This is because of (this)[https://go.dev/blog/loopvar-preview]. We had other parts of the code affected by this before, there was a bug here (see the third commit). 
I don't find the syntax nicer personally, but maybe once this is moved from experimental we can go back to regular ranges.